### PR TITLE
It block assertions

### PR DESCRIPTION
**Pull request by DuncanFenning**
#### What does this PR do? (please provide any background)

This PR re words the best practices for assertions within it-blocks.

This should make things a bit clearer and give more lenience about what should/shouldn't be asserted within 1 it block.

Any recommendations for more improvements would be appreciated.
#### What tests does this PR have?

n/a
#### How can this be tested?

read the alterations and make sure they are understable
#### What gif best describes how you feel about this work?

---
- [x] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.
#### Reviewers

**Review 1**
- [x] :+1:

**Review 2** *
- [x] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

File: testing-standard.md [hunk 1]
```diff
@@ -47,11 +47,30 @@ The location of the tests directory may differ between existing projects, but fo
 Test files should be clean and readable. This means making sensible use of some of Mocha's features to structure test files
 
 ### Using `describe` & `context` blocks
-Mocha's `describe` and `context` syntax can be used to split out tests into logical groups. Typically each function in a module would have its own `describe` block, and various paths through that function would have their own `context` block nested within the parent describe block.
```
Comment (DuncanFenning) [inline testing-standard.md:51]: Just white space..

**Comment by elliottcrush** [comment]
Much clearer. :+1:

File: testing-standard.md [hunk 2]
```diff
@@ -47,11 +47,30 @@ The location of the tests directory may differ between existing projects, but fo
...
+
+Not Recommended
+
+```
+it('returns x, calls y and err equals foo', function(err, result) {
+  expect(err).to.equal('foo');
+  expect(result).to.equal('x');
+  expect(y).to.have.been.called();
+});
+```
```
Comment (msaspence) [inline testing-standard.md:74]: Might want to add a note when multiple assertions might be acceptable e.g. when testing testing multiple aspects of a computationally intensive task.

**Comment by msaspence** [comment]
:ant: Nice

**Pull request merged**